ports/curl/portfile.cmake

```diff
@@ -64,30 +64,36 @@ set(BUILD_OPTIONS
 # Check for ares feature
 if (ares IN_LIST FEATURES)
     message(STATUS "Enabling c-ares")
-    set(BUILD_OPTIONS ${BUILD_OPTIONS} -DENABLE_ARES=ON -DENABLE_THREADED_RESOLVER=OFF)
+    list(APPEND BUILD_OPTIONS -DENABLE_ARES=ON -DENABLE_THREADED_RESOLVER=OFF)
 else ()
-    set(BUILD_OPTIONS ${BUILD_OPTIONS} -DENABLE_ARES=OFF -DENABLE_THREADED_RESOLVER=ON)
+    list(APPEND BUILD_OPTIONS -DENABLE_ARES=OFF -DENABLE_THREADED_RESOLVER=ON)
 endif ()
 
 # Check for ca-bundle feature
 if (ca-bundle IN_LIST FEATURES)
     message(STATUS "Enabling CA bundle")
-    set(BUILD_OPTIONS ${BUILD_OPTIONS} -DCURL_CA_BUNDLE=auto -DCURL_CA_PATH=auto)
+    list(APPEND BUILD_OPTIONS -DCURL_CA_BUNDLE=auto -DCURL_CA_PATH=auto)
 else ()
     message(STATUS "Disabling CA bundle")
-    set(BUILD_OPTIONS ${BUILD_OPTIONS} -DCURL_CA_BUNDLE=none -DCURL_CA_PATH=none)
+    list(APPEND BUILD_OPTIONS -DCURL_CA_BUNDLE=none -DCURL_CA_PATH=none)
 endif ()
 
 # Check for IPV6 feature
 if (ipv6 IN_LIST FEATURES)
     message(STATUS "Enabling IPV6")
-    set(BUILD_OPTIONS ${BUILD_OPTIONS} -DENABLE_IPV6=ON)
+    list(APPEND BUILD_OPTIONS -DENABLE_IPV6=ON)
 else ()
-    set(BUILD_OPTIONS ${BUILD_OPTIONS} -DENABLE_IPV6=OFF)
+    list(APPEND BUILD_OPTIONS -DENABLE_IPV6=OFF)
 endif ()
 
 if (NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME MATCHES "^Windows")
     set(VCPKG_WINDOWS ON)
+
+    # Windows specific features
+    list(APPEND BUILD_OPTIONS
+        -DENABLE_INET_PTON=OFF

```

Comment:
ENABLE_INET_PTON should be ON. We don't need support older Windows version.

Reply:
Good catch! The `option` text for `ENABLE_INET_PTON` is confusing. Will update and merge.